ironfish/src/wallet/wallet.ts

```diff
@@ -394,13 +394,9 @@ export class Accounts {
     })
   }
 
-  /**
-   * Deletes a transaction from the transaction map and updates
-   * the related maps.
-   */
-  async deleteTransaction(transaction: Transaction): Promise<void> {
+  async expireTransaction(transaction: Transaction): Promise<void> {
```

Comment:
I think we should still have a function to delete transactions. I think we'll need it to solve a problem coming up about TX staying around when they are permanently invalid, especially miners fees.

Reply:
Nevermind, ignore this comment I think this is fine.